### DISCUSSION TITLE
Why airflow skips one day for a newly created task?

### DISCUSSION THREAD

**Discussion by bobfang1992** [apache/airflow#13915] [Q&A]
Hi, sorry not sure where is the best place to post this. Apologies if this is not the best venue.

I have a newly created daily dag and I have set it up yesterday (Jan. 25th), once it is loaded by airflow I can see it is run once (scheduled_2021-0124T00:00:00+00:00), and then I manually triggered it once just to see if it works and it did (manual_2021-01-25). 

Now time is 08:24 UTC Jan 26th. But I did not see any run for 01-25. I have used `airflow dags next-execution` and found out airflow is planning to execute the dag for 01-26 directly, possibly on 01-27 00:00 UTC. So it will skip 01-25 entirely.

I am wondering why this behaviour? Is there any reason behind this?

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/SYbEf.png

**Comment by jedcunningham** [comment]
This is a known issue and should be fixed in 2.0.1: #13963

**Reply by bobfang1992** [reply]
Hi thanks! When will the docker images for 2.0.1 be released?

**Reply by jedcunningham** [reply]
The last I saw was February 8th, however, that is subject to change of course: #13909

If you need a solution in the meantime and you don't need a specific `execution_date` you should be able to set it in the past to avoid this issue.
